docs/commands.md

```diff
@@ -1,150 +1,538 @@
+      11) "minHslot"
+      12) (integer) 5461
+      13) "maxHslot"
+      14) (integer) 10922
+```
+
+## RG.PYEXECUTE
+The **RG.PYEXECUTE** command executes a Python [function](functions.md#function).
+
+**Redis API**
+
+```
+RG.PYEXECUTE "<function-definition>" [REQUIREMENTS "<dep> ..."] [UNBLOCKING]
+```
+
+_Arguments_
+
+* _function-definition_: the Python function
+* _REQUIREMENTS_: this subcommand ensures that list of dependencies it is given as an argument is installed on each shard before execution
+* _UNBLOCKING_: doesn't block the client during execution
+
+_Return_
+
+An error is returned if the function can't be parsed, as well as any that are generated by non-RedisGears functions used.
 
-### Returns
-List of registered executions:
-* registrationId
-* execution reader
-* execution description
+When used in `UNBLOCKING` mode reply is an [execution ID](functions.md#executionid).
+
+Any results and errors generated by the function are returned as an array made of two sub-arrays: one for results and the other for errors.
+
+A simple 'OK' string is returned if the function has no output (i.e. it doesn't consist of any functions with the [run](functions.md#run) action).
+
+**Examples**
+
+```
+127.0.0.1:6379> RG.PYEXECUTE "GB().run()"
+1) (empty list or set)
+2) (empty list or set)
+```
+
+## RG.PYSTATS
+The **RG.PYSTATS** command returns memory usage statistics from the [Python interpreter](runtime.md#python-interpreter).
+
+**Redis API**
+
+```
+RG.PYSTATS
+```
+
+_Return_
+
+An array that consists of alternating key name and value entries as follows:
+
+* **TotalAllocated**: a total of all allocations over time in bytes
+* **PeakAllocated**: the peak value of allocations
+* **CurrAllocated**: the currently allocated memory in bytes
+
+**Examples**
+
+```
+127.0.0.1:6379> RG.PYSTATS
+1) "TotalAllocated"
+2) (integer) 113803317
+3) "PeakAllocated"
+4) (integer) 8432603
+5) "CurrAllocated"
+6) (integer) 5745816
+```
+
+## RG.REFRESHCLUSTER
+The **RG.REFRESHCLUSTER** command refreshes the node's view of the cluster's topology.
+
+!!! important "Open Source Redis Cluster"
+    The `RG.REFRESHCLUSTER` command needs to be executed on each of the cluster's nodes.
+
+**Redis API**
+
+```
+RG.REFRESHCLUSTER
+```
+
+_Return_
+
+A simple 'OK' string.
+
+**Examples**
+
+```
+127.0.0.1:6379> RG.REFRESHCLUSTER
+OK
+```
 
 ## RG.UNREGISTER
-```sql
-RG.UNREGISTER registrationId
+The **RG.UNREGISTER** command removes the [registration](functions.md#registration) of a function.
+
+**Redis API**
+
+```
+RG.UNREGISTER <id>
 ```
 
-* registrationId: the registrationId to drop.
+_Arguments_
+
+* **id**: the [registration ID](functions.md#registrationid) for removal
+
+_Return_
 
-### Returns
-`OK` on success or error when:
-* id not exists
-* the reader does not support unregister functionality
+A simple 'OK' string, or an error. An error is returned if the registration ID doesn't exist or if the function's reader doesn't support the unregister operation.
```

Comment:
RG.UNREGISTER reply is sent before the unregister reached to all the shards so in theory you might get the OK reply and the registration might still exists on another shards, I plan to fix it in future releases, should we mentioned it?

Reply:
Nope, just fix it ;)